### DISCUSSION TITLE
Karavan-Installer utility is missing in the new 4.5 version

### DISCUSSION THREAD

**Discussion by bitsInnovate00** [apache/camel-karavan#1248] [Q&A] [answered]
### Ask a question

How can I migrate the below old karavan 4.3 installer configuration to the new 4.5 karavan.
`
$JAVA_HOME/bin/java -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:9251 -jar karavan-installer-4.3.0.jar --git-repository=<http://git-repo> --git-password=<PAT> --git-username=<username> --node-port=30777 --image-registry=docker.io --image-registry-username=<username> --image-registry-password=<docker_pat> --image-group=<group>  --nexus-proxy=true --devmode-image=<group>/camel-karavan-devmode:4.3.0
`
karavan-insaller jar itself is missing . I'm using Kubernetes approach of starting karavan and I'm not sure how can I use the below configuration with the new 4.5 version.

As per the documentation https://github.com/apache/camel-karavan/blob/main/docs/WEB_KUBERNETES.md , its mentioned to change the configuration in secrets.yaml and run the Kustomize command ` kubectl apply -k . ` , but I couldnt find how to configure nexus-proxy in secrets.yaml.  Any thoughts  ?

### Variant

Web Application

### Container Management (if applicable)

Kubernetes

### Operating System (if applicable)

Linux

### Version

4.5.0

**Comment by davsclaus** [comment, marked as answer]
There is no java installer anymore, you install via those kubernetes manifest files as documented in that link.

**Reply by bitsInnovate00** [reply]
How to configure nexus proxy with the kubernetes manifest files. In the case of installer , the property --nexus-proxy was there .

**Reply by mgubaidullin** [reply]
Nexus proxy is not a part of Karavan. Please use Nexus documentation.
